src/AaveV3ListGhoStewards_20230507/GhoAaveV3GhoSteward_20230507_Payload.t.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.16;
+import 'forge-std/Test.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {ProtocolV3TestBase, InterestStrategyValues, ReserveConfig} from 'aave-helpers/ProtocolV3TestBase.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+import {GhoAaveV3GhoSteward} from './GhoAaveV3GhoSteward_20230507_Payload.sol';
+import {IGhoSteward} from 'gho-core/contracts/misc/interfaces/IGhoSteward.sol';
+import {IPoolDataProvider} from 'aave-v3-core/contracts/interfaces/IPoolDataProvider.sol';
+
+import {IGhoDiscountRateStrategy} from 'gho-core/contracts/facilitators/aave/interestStrategy/interfaces/IGhoDiscountRateStrategy.sol';
+import {GhoInterestRateStrategy} from 'gho-core/contracts/facilitators/aave/interestStrategy/GhoInterestRateStrategy.sol';
+import {IGhoToken} from 'gho-core/contracts/gho/interfaces/IGhoToken.sol';
+
+contract GhoAaveV3GhoSteward_20230507_Payload_Test is ProtocolV3TestBase {
+  GhoAaveV3GhoSteward public proposalPayload;
+  address immutable ZERO_ADDRESS = 0x0000000000000000000000000000000000000000;
```

Comment:
you can remove and use `address(0)` where needed instead

Reply:
resolved 3f169bdfd649faeca2ddbfbe92d00350ee5be7c5